gimme_free_epic_shit.js

```diff
@@ -1,50 +1,54 @@
 const { Launcher } = require(`epicgames-client`);
 const { email, password } = require(`${__dirname}/config.json`);
-const epic = new Launcher({
+const defaultPageSize = 100;
+const epicLauncher = new Launcher({
     email: process.argv[2] || email,
     password: process.argv[3] || password
 });
 
+console.log('Trying to log in as ' + epicLauncher.config.email);
```

Comment:
In case of failed attempt to log in, the user will never know which account was attempting to log in. I've tried running the script on 5+ accounts and having this line really helped figuring out which one threw the error.
Maybe we can change the "Logged in" message to just say "Logged in successfully!"?
That way we keep only 1 message with the account email in it, but giving more information in case of failed login attempt.

Reply:
In that case I'd just change the error message to include the login.

Reply:
done